scripts/package.json

```diff
@@ -16,7 +16,7 @@
     "typescript": "^4.2.3"
   },
   "dependencies": {
-    "@dcl/protocol": "^1.0.0-4896874496.commit-2ebeae0",
+    "@dcl/protocol": "https://sdk-team-cdn.decentraland.org/@dcl/protocol/branch//dcl-protocol-1.0.0-4951436628.commit-4280a97.tgz",
```

Comment:
TODO: change protocol version before merging 
(adding this comment so we not forget)